service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/utils/DateFormatUtil.java

```diff
@@ -23,4 +24,14 @@ public static String formatDate(Date date) {
                 .atZone(ZoneId.of(UK_ZONE_ID))
                 .toLocalDateTime());
     }
+
+    public static String formatDate(Instant instant) {
+        if (instant == null) {
```

Comment:
I don't think we should pass null here, you could also use getValue

Reply:
Instant does not have "getValue" method. Do you mean something else?